lib/cli/config.js

```diff
@@ -38,8 +38,7 @@ const isModuleNotFoundError = err =>
  * returns an object (but could throw)
  */
 const parsers = (exports.parsers = {
-  yaml: filepath =>
-    require('js-yaml').safeLoad(fs.readFileSync(filepath, 'utf8')),
+  yaml: filepath => require('js-yaml').load(fs.readFileSync(filepath, 'utf8')),
```

Comment:
What is the reason to change from `safeLoad` to `load`?

Reply:
They changed their api: https://github.com/nodeca/js-yaml/blob/master/migrate_v3_to_v4.md